iXBRLViewerPlugin/viewer/src/less/viewer.less

```diff
@@ -122,7 +122,8 @@ div.ixbrl-table-handle {
     width: 9rem;
     height: initial;
     left: -0.1rem;
-    padding: 2 * @export-handle-size - 0.1rem 0 0 0;
+    padding: 0;
+    z-index: 1;
```

Comment:
I know very little about CSS, so just questions:
1. does `padding: 0` do something different from omitting that line altogether?
2. does `z-index: 1` cause any issues if the input iXBRL is already using `z-index: 1` (or 2, etc.)?

Reply:
1. It doesn't appear to currently, but I think it's worth removing padding explicitly since if padding is re-introduced to this element in any way (higher up the CSS "hierarchy") it will recreate the issue.
2. Sharing `z-index` values is fine, the browser just resorts to document order for layering at that point (all elements share `z-index: 0` by default). Having hover elements "nudged" above the rest can be helpful because it prevents `z-index: 0` elements from earlier in the DOM that might bleed into this element's space and prevent hovering. I can't think of how to create an example from the current viewer but I can imagine it.

Disclaimer: I don't have any real clue about what CSS best practices are beyond the obvious stuff.